### DISCUSSION TITLE
Calibration script

### DISCUSSION THREAD

**Discussion by matijazigic** [ifm/ifm3d#435] [Q&A] [answered]
According to [this](https://ifm3d.com/latest/SoftwareInterfaces/Toolbox/ExtrinsicCalibration/StaticCameraCalibration/README.html#run-the-calibration) documentation there should be a script for calibration named `calib_cam.py`, but it is not located under https://github.com/ifm/documentation.git like stated on that page. Do you know where I can find it?

**Comment by lola-masson** [comment, marked as answer]
Hi @matijazigic, 

The calibration script is in the corresponding subfolder: https://github.com/ifm/documentation/tree/main/SoftwareInterfaces/Toolbox/ExtrinsicCalibration/StaticCameraCalibration
